test.js

```diff
@@ -6,14 +6,15 @@ const redis = require('./index');
 const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
 
 tap.test('createClient', async (t) => {
-  t.type(redis.createClient(), 'RedisClient');
+  let client = redis.createClient();
```

Comment:
`const` would be preferred here since the client variable isn't being modified in this scope

Reply:
fixed